worker/uniter/runner/jujuc/commit.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jujuc
+
+import (
+	"github.com/juju/cmd"
+	jujucmd "github.com/juju/juju/cmd"
+)
+
+// CommitCommand implements the commit command.
+type CommitCommand struct {
+	cmd.CommandBase
+	ctx CommitCommandContext
+}
+
+type CommitCommandContext interface {
+	Commit() error
+}
+
+// NewCommitCommand returns a commit command.
+func NewCommitCommand(ctx Context) (cmd.Command, error) {
+	return &CommitCommand{ctx: ctx}, nil
+}
+
+// Info returns information about the Command.
+// Info implements part of the cmd.Command interface.
+func (c *CommitCommand) Info() *cmd.Info {
+	doc := `
+commit saves current changes made by the charm.  This is usually done if the 
```

Comment:
Perhaps s/usually/automatically would be more informative here?

Also, on the following line s/exist/exits

Reply:
updated.